app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(
+            self, name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+        return None
+
+    def feed(self) -> None:
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
```

Comment:
The `feed` method is defined to return `None`, but it returns `self.appetite` when the animal is fed. This contradicts the method's return type and is unnecessary since the return value is not used. Consider removing the return statement.